server/app/auth/oidc/OidcProfileAdapter.java

```diff
@@ -57,15 +57,26 @@ public OidcProfileAdapter(
 
   protected abstract void adaptForRole(CiviFormProfile profile, ImmutableSet<Roles> roles);
 
+  /** Create a totally new CiviForm profile informed by the provided OidcProfile. */
+  public abstract CiviFormProfile createEmptyCiviFormProfile(OidcProfile profile);
+
+  protected Optional<String> getEmail(OidcProfile oidcProfile) {
```

Comment:
Do we intend for subclasses to be able to override this? If not, can it be marked as final? Same for any other protected and non-final methods in this class.